scripts/ci/run-jenkins.sh

```diff
@@ -94,6 +94,23 @@ if [[ ${CI_TAGS} == 'product-sdks' ]];
 	   exit 0
 fi
 
+if [[ ${CI_TAGS} == 'webassembly' ]];
```

Comment:
should be `${CI_TAGS} == *'webassembly'*` since we want to support multiple CI tags (just saw this applies to product-sdks above too).

Reply:
sounds good, will make the change.